pages/README.md

```diff
@@ -44,14 +47,14 @@
 * [Store Seed](/pages/bitmark-references/bitmark-sdk/store-seed.md)
 * [Web Socket](/pages/bitmark-references/bitmark-sdk/websocket.md)
 
-**[Bitmark\-CLI](/pages/bitmark-references/bitmark-cli/README.md)**
+**[CLI](/pages/bitmark-references/bitmark-cli/README.md)**
 * [Quick Setup](/pages/bitmark-references/bitmark-cli/quick-setup.md)
-* [Bitmark\-CLI Command Reference](/pages/bitmark-references/bitmark-cli/bitmark-cli.md)
+* [Command Reference](/pages/bitmark-references/bitmark-cli/bitmark-cli.md)
 
 <br>
 
-## [Bitmark Appendix](/pages/bitmark-appendix/README.md#bitmark-appendix)
+## [Appendix](/pages/bitmark-appendix/README.md#bitmark-appendix)
 
 * [Bitmark versus Ethereum](/pages/bitmark-appendix/bitmark-eth-comparison.md)
 * [Bitmark shares](/pages/bitmark-appendix/bitmark-shares.md)
-* [Bitmark papers](/pages/bitmark-appendix/bitmark-papers.md)
+* [Papers](/pages/bitmark-appendix/bitmark-papers.md)
```

Comment:
Perhaps "White Papers"?

Reply:
I am using the same wording as on bitmark.com website. I am ok with both Papers & White Papers.  If you think "White Papers" is more reasonable, I will update it.

Reply:
I think so.